numalogic/transforms/_movavg.py

```diff
@@ -86,24 +87,20 @@ class ExpMovingAverage(StatelessTransformer):
     Args:
```

Comment:
I think the class doc has to be updated for this function.

Reply:
+1 on updating class doc

Reply:
Done